README.md

```diff
@@ -0,0 +1 @@
+Run tests only once `npm test -- --single-run`
```

Comment:
I think it would be nice if you expand the README file with some generic package installation instructions and then have a subsection on running tests.